apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-resources/vo-resources-states/vo-resources-states.component.ts

```diff
@@ -14,31 +13,26 @@ export class VoResourcesStatesComponent implements OnInit {
 
   @HostBinding('class.router-component') true;
 
-  constructor(private route: ActivatedRoute,
-              private taskService: TasksManagerService,
-              private customIconService: CustomIconService) { }
+  constructor(private taskService: TasksManagerService,
+              private entityStorageService: EntityStorageService) { }
```

Comment:
in this component in this whole page when it is initially loaded and user want to see the state of some task in resources the table of tasks shows nothing 
![image](https://user-images.githubusercontent.com/32871167/143480353-fd3773e5-f0c4-462b-9591-d48d586b82d4.png)